cmd/cp-url-syntax.go

```diff
@@ -42,7 +43,7 @@ func checkCopySyntax(ctx *cli.Context) {
 	for _, srcURL := range srcURLs {
 		_, _, err := url2Stat(srcURL)
 		if err != nil {
-			fatalIf(err.Trace(srcURL), fmt.Sprintf("Unable to stat '%s'.", srcURL))
+			console.Fatalf("Unable to determine validity of source %s\n", srcURL)
```

Comment:
`Unable to validate source %s`
or
`Unable to check source %s`
or
`Unable to locate source %s`

I think these would be better. I'll approve if you vote for no.